docs/whats_new/v0.4.1.md

```diff
@@ -1,9 +1,191 @@
 # Version 0.4.1
 
-TODO: Add description
+We're happy to announce the AutoGluon 0.4.1 release. 0.4.1 contains minor enhancements to Tabular and Text modules, along with many quality of life improvements and fixes.
+
+This release is **non-breaking** when upgrading from v0.4.0. As always, only load previously trained models using the same version of AutoGluon that they were originally trained on. Loading models trained in different versions of AutoGluon is not supported.
+
+This release contains [**29** commits from **10** contributors](https://github.com/awslabs/autogluon/graphs/contributors?from=2022-03-10&to=2022-05-23&type=c)!
```

Comment:
29 is not the correct number of commits. I see 56: https://github.com/awslabs/autogluon/compare/v0.4.0...master

Reply:
Correct. 55: one is just a bump of version PR